packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx

```diff
@@ -62,6 +64,7 @@ export const ShowPageRightContainer = ({
   tasks,
   notes,
   emails,
+  charts,
```

Comment:
I feel like we would have do this if each entity had multiple charts attached to it, which is probably somewhere we want to get ("dynamic chart filters",  where personId = xxx). But for now a chart is more a workspace level object so I would have seen something like this instead to divert the show page https://github.com/twentyhq/twenty/pull/6219/files#diff-78f7e04001929f7fe59dc44e1320f0e56b1bf85f1c05905f9c6d1415cba76888R267